src/main.rs

```diff
@@ -48,6 +49,7 @@ pub struct SnsCanisterIds {
     pub governance_canister_id: CanisterId,
     pub ledger_canister_id: CanisterId,
     pub root_canister_id: CanisterId,
+    pub dapp_canister_id_list: Vec<CanisterId>,
```

Comment:
Please update the example on the clap flag to include this new field in the json